api/tests/unit/domain/usecases/create-organizations-with-tags-and-target-profiles_test.js

```diff
@@ -484,31 +482,4 @@ describe('Unit | UseCase | create-organizations-with-tags-and-target-profiles',
     // then
     expect(organizationInvitationService.createProOrganizationInvitation).to.not.have.been.called;
   });
```

Comment:
Et suivant la même idée, j'ai été gêné par le test `validation error cases - when there is more than one occurrence of the same organization in data file` qui me fait penser qu'il n'est pas valide de créer des organisations avec les mêmes `externalId`.

Reply:
J'ai retiré ce test.